### ISSUE TITLE
Unable to change Bin Width with keyboard 

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/projectile-data-lab#256]
**Test device**
MacBook Air M1 chip

**Operating System**
14.3.1

**Browser**
Safari 17.3.1

**Problem description**
For https://github.com/phetsims/qa/issues/1060 on all screens: I am unable to move through the items in the combobox with keyboard nav.

**Steps to reproduce**
1. On any screen, tab to the Bin Width Combobox and press enter--the pink focus box should be around the entire listbox but isn't
2. Try to use arrow keys to select a different value

**Visuals**
<img width="193" alt="Screenshot 2024-03-18 at 4 14 00 PM" src="https://github.com/phetsims/projectile-data-lab/assets/87318828/e4255fbb-10f6-4b61-83dd-9018fc36eaed">

<details>

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Projectile Data Lab‬
URL: https://phet-dev.colorado.edu/html/projectile-data-lab/1.0.0-dev.22/phet/projectile-data-lab_all_phet.html
Version: 1.0.0-dev.22 2024-03-18 16:30:54 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.3.1 Safari/605.1.15
Language: en-US
Window: 1393x699
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by KatieWoe** [comment]
This may be platform specific. This works on Win 11 Chrome for me.

**Comment by Nancy-Salpepi** [comment]
It definitely seems to be specific to safari. It works with mac + chrome and with the Chromebook. 
I am on safari 17.3.1

**Comment by Nancy-Salpepi** [comment]
This seems to be a general issue with combobox and mac + safari. I have the same problem on main with MSS and Kepler's. Things work fine with published. 
Here is the issue: https://github.com/phetsims/sun/issues/878

**Comment by Nancy-Salpepi** [comment]
This issue has been fixed in https://github.com/phetsims/sun/issues/878 🎉. 
Closing.
